bql/grammar/grammar_test.go

```diff
@@ -51,6 +51,7 @@ func TestAcceptByParse(t *testing.T) {
 		`select ?a from ?b where{?s ?p ?o as ?x id ?y type ?z};`,
 		`select ?a from ?b where{?s ?p ?o as ?x type ?y id ?z at ?t};`,
 		`select ?a from ?b where{?s ?p ?o as ?x id ?y type ?z at ?t};`,
+		`select ?a from ?b where{?s ?p ?o at ?x};`,
```

Comment:
Could you add a test to the planner as well?

Reply:
Sure. It was added with the [PR 135](https://github.com/google/badwolf/pull/135) that requires this kind of `AT` binding to work. 😀